docs/releases/2.11.3.rst

```diff
@@ -0,0 +1,43 @@
+============================
+Wagtail 2.11.3 release notes
+============================
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+What's new
+==========
+
+Bug fixes
+~~~~~~~~~
+
+ *
+
+
+Upgrade considerations
```

Comment:
I don't usually look in point releases for upgrade considerations, would it be worth adding a link to this from the 2.11 upgrade considerations for those upgrading later?

Reply:
Yep, I agree.